portfolio/src/main/java/com/google/sps/data/CommentHistory.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class representing the subtraction game, where players take turns subtracting from 21 to reach 0.
+ *
+ * <p>Note: The private variables in this class are converted into JSON.
+ */
+public class CommentHistory {
+    private long id;
+
+  /** List of descriptions of turns, e.g. "Player 1 took 3. New total: 18" */
+  private String name;
+  private String comment;
+  private long timestamp;
+  
+  public CommentHistory()
```

Comment:
Unless you intend to create a CommentHistory() without an ID, name, comment, and timestamp, it's probably better to delete this empty constructor.